src/ASM/LR/ASMu2Dmx.C

```diff
@@ -1073,7 +1073,8 @@ bool ASMu2Dmx::refine (const LR::RefineData& prm, Vectors& sol)
     for (size_t j = 0; j < m_basis.size(); j++) {
       Vector bVec;
       this->extractNodeVec(solvec, bVec, 0, j+1);
-      LR::extendControlPoints(m_basis[j].get(), bVec, nfx[j]);
+      if (ASMmxBase::Type != ASMmxBase::SUBGRID || j != 0)
```

Comment:
A `size_t` can never be negative so `j > 0`would make more sense here?

Reply:
sure.